dogfood/main.tf

```diff
@@ -194,6 +197,7 @@ resource "coder_agent" "dev" {
     elif [ -f ~/personalize ]; then
       echo "~/personalize is not executable, skipping..." | tee -a ~/.personalize.log
     fi
+    cd ${data.coder_parameter.repo_dir.value}
```

Comment:
To get the same behavior I added cd ${data.coder_parameter.repo_dir.value} as the last line in startup_script

Reply:
removing it.